api/proto/info/info.proto

```diff
@@ -183,6 +183,13 @@ enum HwSecurityModuleStatus {
     ENABLED  = 3;  //HSM is found and being actively used
 }
 
+enum DeviceEncryptionStatus {
```

Comment:
I don't see how a single enum can capture the fact that the different objects will be encrypted and at different levels.